pkgs/applications/misc/merkaartor/default.nix

```diff
@@ -13,24 +28,49 @@ mkDerivation rec {
   };
 
   patches = [
+    # Fix build with Qt 5.15 (missing QPainterPath include)
     (fetchpatch {
       url = "https://github.com/openstreetmap/merkaartor/commit/e72553a7ea2c7ba0634cc3afcd27a9f7cfef089c.patch";
       sha256 = "NAisplnS3xHSlRpX+fH15NpbaD+uM57OCsTYGKlIR7U=";
     })
+  ] ++ lib.optionals withGpsdlib [
```

Comment:
Can we apply this unconditionally to prevent bit rotting?

Reply:
Done